Jetcaster/app/src/main/java/com/example/jetcaster/ui/home/Home.kt

```diff
@@ -174,7 +188,15 @@ fun Home(
     onCategorySelected: (Category) -> Unit,
     navigateToPlayer: (String) -> Unit,
     onTogglePodcastFollowed: (String) -> Unit,
+    onLibraryPodcastSelected: (Podcast?) -> Unit
 ) {
+    // Effect that changes the home category selection when there are no subscribed podcasts
+    LaunchedEffect(key1 = featuredPodcasts) {
+        if (featuredPodcasts.isEmpty()) {
+            onHomeCategorySelected(HomeCategory.Discover)
+        }
+    }
+
```

Comment:
I think this should just be in the viewmodel where you combine your state

Reply:
Good call